com.unity.robotics.urdf-importer/Tests/Runtime/MeshProcessing/StlAssetPostProcessorTests.cs

```diff
@@ -0,0 +1,51 @@
+using System.IO;
+using System.Collections.Generic;
+using NUnit.Framework;
+using UnityEditor;
+using UnityEngine;
+
+namespace Unity.Robotics.UrdfImporter.Tests
+{
+    public class StlAssetPostProcessorTests
+    {
+        const string k_AssetRoot = "Assets/Tests/Runtime/StlAssetPostProcessorTests";
+        const string k_StlCubeSourcePath = "Packages/com.unity.robotics.urdf-importer/Tests/Runtime/Assets/URDF/cube/meshes/cube.stl";
+        string m_StlCubeCopyPath;
+
+        [SetUp]
+        public void SetUp()
+        {
+            m_StlCubeCopyPath = k_AssetRoot + "/cube.stl";
+            RuntimeURDF.SetRuntimeMode(false);
+            Directory.CreateDirectory(k_AssetRoot);
+        }
+        
+        [Test]
+        public void StlPostprocess_NewStl_DontCreatePrefab()
```

Comment:
We can use a name akin to a title. For eg: "StlAssetCreatationTest". And method names are always in PascalCase.

Reply:
For unit test names, we are following the [.NET unit test best practices ](https://docs.microsoft.com/en-us/dotnet/core/testing/unit-testing-best-practices#better), i.e. `WhatIsTested_Scenario_ExpectedResult()`